golos.charge/golos.charge.hpp

```diff
@@ -35,21 +35,21 @@ class charge : public contract {
 
 private:
     struct balance {
-        symbol charge_symbol;
+        uint64_t charge_symbol;
```

Comment:
Why do you change type from `symbol` to `uint64_t`?

Reply:
The main problem is that the user in the database will see a number that is difficult to interpret.

Reply:
I agree that symbol as the primary key would be convenient. For now, I’m going to use charge_symbol as a key, adding two redundant fields: symbol_code and uint8_t.